File: Easy/771. Jewels and Stones/solution.py
class Solution(object):
    def numJewelsInStones(self, jewels, stones):
        """
        :type jewels: str
        :type stones: str
        :rtype: int
        """
        se = set([x for x in jewels])
        out = 0
        for char in stones:
            if char in se :
                out += 1
            else : continue
        return out

File: Easy/771. Jewels and Stones/solution.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        Set<Character> set = new HashSet<>();
        for (int i = 0; i<jewels.length(); i++){
            if (set.contains(jewels.charAt(i))){
                continue;
            } else {
                set.add(jewels.charAt(i));
            } 
        }
        int out = 0;
        for (int i = 0; i<stones.length(); i++){
            if (set.contains(stones.charAt(i))){
                out++;
            } else {
                continue;
            } 
        }
        return out;   
    }
}